DOCS.md

```diff
@@ -188,6 +188,10 @@ vars
 : a map of variables to pass to the Terraform `plan` and `apply` commands.
 Each value is passed as a `-var <key>=<value>` option.
 
+var_files
+: a list of variable files to pass to the Terraform `plan` and `apply` commands.
```

Comment:
Says it is for the `plan` and `apply` command but seems you only made changes to have it work with `plan`.

Please either update the documentation or add the additional required code for `apply`.